roles/pulp_common/tasks/main.yml

```diff
@@ -32,6 +32,12 @@
   loop_control:
     loop_var: __pulp_common_req_var
 
+- name: Check required roles if FIPS detected
+  assert:
+    that:
+      - "'pulp_devel' in role_names"
+  when: ansible_fips
```

Comment:
Previously I was suggesting that we remove the setting of `ansible_fips` altogether, but what I'm seeing now is this is very close to a feature that fails very early if you're trying to install without the dev role at all. That actually sounds pretty sweet. To get to that fully, I think all that's needed is to put an error message here identifying that the Pulp installer cannot install into FIPS environments. @gerrod3 wdyt?